tests/integration/routes/test_questionnaire.py

```diff
@@ -44,3 +44,12 @@ def test_head_request_on_block_with_optional_date_answer(self):
         )
         self.head("/questionnaire/date-non-mandatory-block/")
         self.assertStatusOK()
+
+    def test_options_request_exits_before_request_early(self):
+        self.launchSurvey("test_hub_and_spoke")
+        with self.assertLogs() as cm:
```

Comment:
I think the naming here could be improved, `cm` is a bit vague - maybe `logs`?

Reply:
Done ✅